x-pack/plugins/uptime/public/components/monitor/ping_list/columns/ping_timestamp.tsx

```diff
@@ -152,9 +149,9 @@ export const PingTimestamp = ({ timestamp, ping }: Props) => {
       ) : (
         <EuiFlexGroup gutterSize="s" alignItems="center">
           <EuiFlexItem>
-            <NoImageAvailable />
+            <EuiLoadingSpinner size="xl" />
```

Comment:
I'm not sure this was the right approach, so I'd love your feedback @shahzad31 and @justinkambic. We originally had a box that said 'Image Not Available'. Design is requesting a spinner for the loading state, but do we distinguish between loading and unavailable at all? Should I attempt to derive loading from `useFetcher` and only display the spinner in that scenario, and display 'Image Not Available' if `loaded && !imgSrc`.

Reply:
Also, this triggered a snapshot change. I am leaving this PR in draft state until I have the opportunity to refactor this test away from snapshots.

Reply:
i think using status from useFetcher is probably a better approach.